packages/tokens/BricomanProperties/color/heading.json

```diff
@@ -0,0 +1,14 @@
+{
+  "color": {
+    "heading": {
+      "line": {
```

Comment:
@tiloyi Can we change `line` to `underline` without impact? If not, it's OK, and we can keep `line` as well 🙂

Reply:
@caqueste it's fixed :)